packages/front-end/pages/datasources/[did].tsx

```diff
@@ -260,123 +207,36 @@ mixpanel.init('YOUR PROJECT TOKEN', {
               </p>
 
               <div className="card py-3 px-3 mb-4">
-                {/* TODO: design changes for Identity Joins nested */}
-                {/* region Identifier Types */}
                 <DataSourceInlineEditIdentifierTypes
                   onSave={updateDataSource}
-                  onCancel={cancelUpdateDataSource}
+                  onCancel={() => undefined}
```

Comment:
`onCancel` and `onSave` are exposed, but since each section manages its own modal, `onCancel` is not used. We can delete this if it makes sense to, but it would require some duplication of props. We may want to fire a callback when an inner modal is closed, for that reason I left it, but if we'd rather not have this, let me know and I can re-work the props. It will include some props duplication and not being able to use the existing shared props.